components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -290,6 +290,10 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
     }
   }
 
+  getConfirmBtnText(): string {
+    return this.saving ? 'Saving Rule...' : 'Save Rule';
+  }
+
```

Comment:
Small ternary returning the desired text.

Reply:
Question for @susanev: For tokens, projects, and teams we use just "Save" and "Saving...". Is this one different for a reason?

Reply:
@msorens yes intentional since we are in this full-page design and trying to orient users into what they are saving.